OIIInspector/image_manager.py

```diff
@@ -0,0 +1,198 @@
+import logging
+from retry import retry
+import subprocess
+import time
+import shlex
+from OIIInspector.exceptions import AddressAlreadyInUse, OIIInspectorError
+from OIIInspector.utils import run_cmd
+
+log = logging.getLogger(__name__)
+
+
+class ImageManager:
+    """
+    Context manager for working wight images
+    """
+    _grpc_start_port = 50051
+    _grpc_max_port_tries = 100
+    _grpc_init_wait_time = 5
+    _base_container_name = "OIIInspector_running_image"
+    _image_is_running = False
+    _container_platform = None
+
+    def __init__(self, image_address):
+        """
+        Initialization of the ImageManager
+
+        :param image_address:
+        :type image_address:
+        """
+        self._get_container_platform()
```

Comment:
I would move this outside of constructor, probably to start_image. Running code like this during object construction is not good practice. 
When you move it, could you make sure that it only runs the commands if the value is not yet known?